components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -811,12 +811,17 @@ define([
                     this.getAgeLabel().show();
                     labels.push(this.getAgeLabel());
                 }
+                this.getExternalIDLabel() && labels.push(this.getExternalIDLabel());
+                if (this.getCommentsLabel()) {
+                    this.getCommentsLabel().show();
+                    labels.push(this.getCommentsLabel());
+                }
             } else {
                 this.getNameLabel() && this.getNameLabel().hide();
                 this.getAgeLabel() && this.getAgeLabel().hide();
+                this.getExternalIDLabel() && labels.push(this.getExternalIDLabel());
```

Comment:
Then questions to discuss: 
 1. Do we need to hide external id in anonimized pdf?
 2. Can we change the order of labels to make external id the first?